tests/virt/node/general/test_windows_vtpm_bitlocker.py

```
@@ -48,9 +48,7 @@ def _wait_encryption_finish(vm):
         try:
             for sample in sampler:
                 if sample:
-                    if all([
-                        True if msg in sample[0] else False for msg in ["100.0%", "Fully Encrypted", "Protection On"]
-                    ]):
+                    if all([msg in sample[0] for msg in ["100.0%", "Fully Encrypted", "Protection On"]]):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)